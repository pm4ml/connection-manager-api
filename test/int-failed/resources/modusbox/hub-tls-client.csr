-----BEGIN CERTIFICATE REQUEST-----
MIIFfzCCA2cCAQAwgbIxCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJXQTEQMA4GA1UE
BxMHU2VhdHRsZTERMA8GA1UEChMITW9kdXNib3gxFDASBgNVBAsTC0VuZ2luZWVy
aW5nMR0wGwYDVQQDExRodWIuZGV2Lm1vZHVzYm94LmNvbTE8MDoGCSqGSIb3DQEJ
AQwtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWluQGh1Yi5kZXYubW9kdXNib3guY29t
MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA9aGgGgJkvv2JfWzvaei4
5+Vf7VicajqvEjqO/KjWQwZLUXRAq3shIm2onN67ftDWj5uo8k2CsR8l/bNPeML8
THyfq5OHrV1i/BX8V4/Dy82caOUnUoq0Rvl7daNSz7McrA+hvMjWD7r9FN8qO4o7
nGxLmZ6AMlThVwTcr8UGR9Z5tKC4RjhHljH5NTSWchmLZj+kQohnK1Gz9zT3ZRhA
ynEkv7jH2oi4YCJNMu/yKgCaUSH11JU+eHJTXePNqKslMNyAPR71aAkGyqJaOPHo
i833m9EkDTSPg+dJTzzW/y8/T7o+kvucFLoF+bpQ1LA8EgY4Z5jAxB9Hy7Nns+z+
Sw74S4+ad5KrXcqwdpHe+uhhRRBBC1bF4UHLQ5+kpvCY8is+KNUCcki8NFb0rh0L
BNK3vbU/a5iUlKCDmebtmB20TEfrLqGdS2b+CvNPDzHDTBt6m61FiA8M1WoWvzgE
8cjYyEG2/lZIQHJ6nzAJ0kFcal8mu7OibkQMEnxHQmsxMi4/NQ4i+SVSWqVb+n3o
YMSLvfOxe8kXYpnOulbbG87ocFL6Y/6ceRYHo9vLVxSVmuS6UOTbMF2JAFWRbkqI
RAOkEeipxvkHy6GLz9UhVMqxK4OmxQ1bLI/Gpk3ULPhBx1lXtyRFeBfk1YjGZlFp
sT9ar3pGdPcRCa0feFkUzOMCAwEAAaCBhjCBgwYJKoZIhvcNAQkOMXYwdDByBgNV
HREEazBpghVodWIxLmRldi5tb2R1c2JveC5jb22CFWh1YjIuZGV2Lm1vZHVzYm94
LmNvbYEtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWluQGh1Yi5kZXYubW9kdXNib3gu
Y29thwSjCgUYhwSjCgUVMA0GCSqGSIb3DQEBDQUAA4ICAQDzGBiYTjwgXnu1+y7y
8HRKLny0EYMpVOrAf/sSrVKsh2ExS3o10VzV+KyewPsSDfew/S7VDxWqZcJlZrEJ
XW/ZgZGkDjsVTgCSMQ/IVPDXKyQIEzCWB8Ne5UnC9oIjUlB5l3Svld4Q0DtS6hE6
3azz8TIYSivwIE8OG/TXQxqtjjJMC2sCcHA9KZsZZwes1vezXR3LIy4R8GBLYpVV
DGiQnNsXnIqcx2nH3kP/l00PK9Kk/jOJxKGbQbs79ntaAJ9AXuQHYxg13q4ppGG+
SCWkEx3LDQTPi+WWhlcETnpvkT1jAH1SWV3ld8c7TcXPpcj0DYTaBYovL6baeCCD
PDObXFU4A0QFivwgvrKicc85JWtg4Y10UCcPRfAxpnT8afV9KY+sX6oZo1cKcCWL
+SDc3Ikyd3Vf3nPvDy/ZqyGx/h5UpFKwgG9usYEzz57LVBxEnaOfRLN2E6sNvuse
rr1yejFQXRqxBPj0PniiNyRy+dsFeCIflAa/cmzYch/lWgpM8pEj4XsYSu7lRMqx
ihmpmqpClZYx0Mu4CTaHW85S0aV6EhMOZgJmRnm+R+QNB4Q44ssnfyeb9TipKCE0
kMCix1EqPoucgIekQzedwe32wDR0oGYMOdA1w44maDC9QaIKLjhu2f+8m+CqeD4Q
vr97InCotkF5yL/eQtiSkUhUjg==
-----END CERTIFICATE REQUEST-----
