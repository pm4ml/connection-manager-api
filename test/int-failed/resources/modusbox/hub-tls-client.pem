-----BEGIN CERTIFICATE-----
MIIGgTCCBGmgAwIBAgIUERbavNDO8LfeDTS0jRaQ7Q6Fj6cwDQYJKoZIhvcNAQEL
BQAwVTELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAldBMRAwDgYDVQQHEwdTZWF0dGxl
MREwDwYDVQQKEwhNb2R1c2JveDEUMBIGA1UECxMLRW5naW5lZXJpbmcwHhcNMTkw
NjA1MjM1NTAwWhcNMjkwNjAyMjM1NTAwWjCBsjELMAkGA1UEBhMCVVMxCzAJBgNV
BAgTAldBMRAwDgYDVQQHEwdTZWF0dGxlMREwDwYDVQQKEwhNb2R1c2JveDEUMBIG
A1UECxMLRW5naW5lZXJpbmcxHTAbBgNVBAMTFGh1Yi5kZXYubW9kdXNib3guY29t
MTwwOgYJKoZIhvcNAQkBDC1jb25uZWN0aW9uLW1hbmFnZXItYWRtaW5AaHViLmRl
di5tb2R1c2JveC5jb20wggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQD1
oaAaAmS+/Yl9bO9p6Ljn5V/tWJxqOq8SOo78qNZDBktRdECreyEibaic3rt+0NaP
m6jyTYKxHyX9s094wvxMfJ+rk4etXWL8FfxXj8PLzZxo5SdSirRG+Xt1o1LPsxys
D6G8yNYPuv0U3yo7ijucbEuZnoAyVOFXBNyvxQZH1nm0oLhGOEeWMfk1NJZyGYtm
P6RCiGcrUbP3NPdlGEDKcSS/uMfaiLhgIk0y7/IqAJpRIfXUlT54clNd482oqyUw
3IA9HvVoCQbKolo48eiLzfeb0SQNNI+D50lPPNb/Lz9Puj6S+5wUugX5ulDUsDwS
BjhnmMDEH0fLs2ez7P5LDvhLj5p3kqtdyrB2kd766GFFEEELVsXhQctDn6Sm8Jjy
Kz4o1QJySLw0VvSuHQsE0re9tT9rmJSUoIOZ5u2YHbRMR+suoZ1LZv4K808PMcNM
G3qbrUWIDwzVaha/OATxyNjIQbb+VkhAcnqfMAnSQVxqXya7s6JuRAwSfEdCazEy
Lj81DiL5JVJapVv6fehgxIu987F7yRdimc66VtsbzuhwUvpj/px5Fgej28tXFJWa
5LpQ5NswXYkAVZFuSohEA6QR6KnG+QfLoYvP1SFUyrErg6bFDVssj8amTdQs+EHH
WVe3JEV4F+TViMZmUWmxP1qvekZ09xEJrR94WRTM4wIDAQABo4HqMIHnMA4GA1Ud
DwEB/wQEAwIFoDATBgNVHSUEDDAKBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB0G
A1UdDgQWBBRu+mqDejGqPwBoW3t+B3Mb560VPDAfBgNVHSMEGDAWgBQJP5EtJFht
yMYkyJ+NSnYb3qsl+DByBgNVHREEazBpghVodWIxLmRldi5tb2R1c2JveC5jb22C
FWh1YjIuZGV2Lm1vZHVzYm94LmNvbYEtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWlu
QGh1Yi5kZXYubW9kdXNib3guY29thwSjCgUYhwSjCgUVMA0GCSqGSIb3DQEBCwUA
A4ICAQBjOEMGovGVIWdjRnzTL9qCFZheni48DKe95S9eNQwp5BEtLzyHxkdnEsZb
M3e55kdknRc4HwtoNabiazhnk6hCBqQNAeI3JxagdMf7HPWnPb2QtTgX9hltPgDz
zWsbAlCJUrlZeMNmyT8yOAtCvdUVUMMTepiEMJwvavNIT5bwdXvd19HZU07YOg5s
btj+PGW+sOuGodPhd29QRe3oP8GXwkxSVSM+YWgUsbQq3rOxVhMEWVHnBejX01D2
/eAk1mVmgF63GgvyboYymdnc/1mNJZNJUOBnk9vSENmDVCIRGrql1g2tQHHtfwnj
rFcDDmc+Nek07zUz+C7K38XiqXOS33BrVxh8caiNC8aSfi6LLrhvWgXbsKBc/UVI
/OzfRw9i70uOAH7Ys+RBxOiy2QKx4P9oOygARlxUpz5tHVcSw14njMfBOghqSLO5
Zcbzr3JF21AgswYSJ2w4PYE9F3poCpdXfffD7bjEbKJVY0cIAqGMyL8pJY6YUt75
fXBbasHXSXAH4mxTHQg3U7Fj+k3qlPHu0yHRXmKbdbFdlP4ywgVrhzS+EunmeXWK
vB6A8l+1ADXq5HydhRwdTkfZm7uPql4xAHqyXRPEVu9tSElutMS7vo2a97qaTK6X
zaoGbLdioYWImIWsl86zeBQ67TwKYt5fHke9mCx53G6/RVmEzQ==
-----END CERTIFICATE-----
