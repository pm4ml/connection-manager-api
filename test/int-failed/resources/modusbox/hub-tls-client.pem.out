Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            11:16:da:bc:d0:ce:f0:b7:de:0d:34:b4:8d:16:90:ed:0e:85:8f:a7
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, ST=WA, L=Seattle, O=Modusbox, OU=Engineering
        Validity
            Not Before: Jun  5 23:55:00 2019 GMT
            Not After : Jun  2 23:55:00 2029 GMT
        Subject: C=US, ST=WA, L=Seattle, O=Modusbox, OU=Engineering, CN=hub.dev.modusbox.com/emailAddress=connection-manager-admin@hub.dev.modusbox.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:f5:a1:a0:1a:02:64:be:fd:89:7d:6c:ef:69:e8:
                    b8:e7:e5:5f:ed:58:9c:6a:3a:af:12:3a:8e:fc:a8:
                    d6:43:06:4b:51:74:40:ab:7b:21:22:6d:a8:9c:de:
                    bb:7e:d0:d6:8f:9b:a8:f2:4d:82:b1:1f:25:fd:b3:
                    4f:78:c2:fc:4c:7c:9f:ab:93:87:ad:5d:62:fc:15:
                    fc:57:8f:c3:cb:cd:9c:68:e5:27:52:8a:b4:46:f9:
                    7b:75:a3:52:cf:b3:1c:ac:0f:a1:bc:c8:d6:0f:ba:
                    fd:14:df:2a:3b:8a:3b:9c:6c:4b:99:9e:80:32:54:
                    e1:57:04:dc:af:c5:06:47:d6:79:b4:a0:b8:46:38:
                    47:96:31:f9:35:34:96:72:19:8b:66:3f:a4:42:88:
                    67:2b:51:b3:f7:34:f7:65:18:40:ca:71:24:bf:b8:
                    c7:da:88:b8:60:22:4d:32:ef:f2:2a:00:9a:51:21:
                    f5:d4:95:3e:78:72:53:5d:e3:cd:a8:ab:25:30:dc:
                    80:3d:1e:f5:68:09:06:ca:a2:5a:38:f1:e8:8b:cd:
                    f7:9b:d1:24:0d:34:8f:83:e7:49:4f:3c:d6:ff:2f:
                    3f:4f:ba:3e:92:fb:9c:14:ba:05:f9:ba:50:d4:b0:
                    3c:12:06:38:67:98:c0:c4:1f:47:cb:b3:67:b3:ec:
                    fe:4b:0e:f8:4b:8f:9a:77:92:ab:5d:ca:b0:76:91:
                    de:fa:e8:61:45:10:41:0b:56:c5:e1:41:cb:43:9f:
                    a4:a6:f0:98:f2:2b:3e:28:d5:02:72:48:bc:34:56:
                    f4:ae:1d:0b:04:d2:b7:bd:b5:3f:6b:98:94:94:a0:
                    83:99:e6:ed:98:1d:b4:4c:47:eb:2e:a1:9d:4b:66:
                    fe:0a:f3:4f:0f:31:c3:4c:1b:7a:9b:ad:45:88:0f:
                    0c:d5:6a:16:bf:38:04:f1:c8:d8:c8:41:b6:fe:56:
                    48:40:72:7a:9f:30:09:d2:41:5c:6a:5f:26:bb:b3:
                    a2:6e:44:0c:12:7c:47:42:6b:31:32:2e:3f:35:0e:
                    22:f9:25:52:5a:a5:5b:fa:7d:e8:60:c4:8b:bd:f3:
                    b1:7b:c9:17:62:99:ce:ba:56:db:1b:ce:e8:70:52:
                    fa:63:fe:9c:79:16:07:a3:db:cb:57:14:95:9a:e4:
                    ba:50:e4:db:30:5d:89:00:55:91:6e:4a:88:44:03:
                    a4:11:e8:a9:c6:f9:07:cb:a1:8b:cf:d5:21:54:ca:
                    b1:2b:83:a6:c5:0d:5b:2c:8f:c6:a6:4d:d4:2c:f8:
                    41:c7:59:57:b7:24:45:78:17:e4:d5:88:c6:66:51:
                    69:b1:3f:5a:af:7a:46:74:f7:11:09:ad:1f:78:59:
                    14:cc:e3
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage: 
                TLS Web Client Authentication
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Subject Key Identifier: 
                6E:FA:6A:83:7A:31:AA:3F:00:68:5B:7B:7E:07:73:1B:E7:AD:15:3C
            X509v3 Authority Key Identifier: 
                keyid:09:3F:91:2D:24:58:6D:C8:C6:24:C8:9F:8D:4A:76:1B:DE:AB:25:F8

            X509v3 Subject Alternative Name: 
                DNS:hub1.dev.modusbox.com, DNS:hub2.dev.modusbox.com, email:connection-manager-admin@hub.dev.modusbox.com, IP Address:163.10.5.24, IP Address:163.10.5.21
    Signature Algorithm: sha256WithRSAEncryption
         63:38:43:06:a2:f1:95:21:67:63:46:7c:d3:2f:da:82:15:98:
         5e:9e:2e:3c:0c:a7:bd:e5:2f:5e:35:0c:29:e4:11:2d:2f:3c:
         87:c6:47:67:12:c6:5b:33:77:b9:e6:47:64:9d:17:38:1f:0b:
         68:35:a6:e2:6b:38:67:93:a8:42:06:a4:0d:01:e2:37:27:16:
         a0:74:c7:fb:1c:f5:a7:3d:bd:90:b5:38:17:f6:19:6d:3e:00:
         f3:cd:6b:1b:02:50:89:52:b9:59:78:c3:66:c9:3f:32:38:0b:
         42:bd:d5:15:50:c3:13:7a:98:84:30:9c:2f:6a:f3:48:4f:96:
         f0:75:7b:dd:d7:d1:d9:53:4e:d8:3a:0e:6c:6e:d8:fe:3c:65:
         be:b0:eb:86:a1:d3:e1:77:6f:50:45:ed:e8:3f:c1:97:c2:4c:
         52:55:23:3e:61:68:14:b1:b4:2a:de:b3:b1:56:13:04:59:51:
         e7:05:e8:d7:d3:50:f6:fd:e0:24:d6:65:66:80:5e:b7:1a:0b:
         f2:6e:86:32:99:d9:dc:ff:59:8d:25:93:49:50:e0:67:93:db:
         d2:10:d9:83:54:22:11:1a:ba:a5:d6:0d:ad:40:71:ed:7f:09:
         e3:ac:57:03:0e:67:3e:35:e9:34:ef:35:33:f8:2e:ca:df:c5:
         e2:a9:73:92:df:70:6b:57:18:7c:71:a8:8d:0b:c6:92:7e:2e:
         8b:2e:b8:6f:5a:05:db:b0:a0:5c:fd:45:48:fc:ec:df:47:0f:
         62:ef:4b:8e:00:7e:d8:b3:e4:41:c4:e8:b2:d9:02:b1:e0:ff:
         68:3b:28:00:46:5c:54:a7:3e:6d:1d:57:12:c3:5e:27:8c:c7:
         c1:3a:08:6a:48:b3:b9:65:c6:f3:af:72:45:db:50:20:b3:06:
         12:27:6c:38:3d:81:3d:17:7a:68:0a:97:57:7d:f7:c3:ed:b8:
         c4:6c:a2:55:63:47:08:02:a1:8c:c8:bf:29:25:8e:98:52:de:
         f9:7d:70:5b:6a:c1:d7:49:70:07:e2:6c:53:1d:08:37:53:b1:
         63:fa:4d:ea:94:f1:ee:d3:21:d1:5e:62:9b:75:b1:5d:94:fe:
         32:c2:05:6b:87:34:be:12:e9:e6:79:75:8a:bc:1e:80:f2:5f:
         b5:00:35:ea:e4:7c:9d:85:1c:1d:4e:47:d9:9b:bb:8f:aa:5e:
         31:00:7a:b2:5d:13:c4:56:ef:6d:48:49:6e:b4:c4:bb:be:8d:
         9a:f7:ba:9a:4c:ae:97:cd:aa:06:6c:b7:62:a1:85:88:98:85:
         ac:97:ce:b3:78:14:3a:ed:3c:0a:62:de:5f:1e:47:bd:98:2c:
         79:dc:6e:bf:45:59:84:cd
